src/solvers/slepc_eigen_solver.C

```diff
@@ -970,21 +989,14 @@ void SlepcEigenSolver<T>::set_initial_space(NumericVector<T> & initial_space_in)
 #if SLEPC_VERSION_LESS_THAN(3,1,0)
   libmesh_error_msg("SLEPc 3.1 is required to call EigenSolver::set_initial_space()");
 #else
-  this->init();
-
-  PetscErrorCode ierr = 0;
-
   // Make sure the input vector is actually a PetscVector
   PetscVector<T> * initial_space_petsc_vec =
-    dynamic_cast<PetscVector<T> *>(&initial_space_in);
+  dynamic_cast<PetscVector<T> *>(&initial_space_in);
```

Comment:
I see this is just a whitespace change, but while you are at it, you may as well change this to a `cast_ptr` call and drop the following `libmesh_error_msg_if` line, since the error check is then done internally. So the whole thing would be in one line:
```
_initial_space = cast_ptr<PetscVector<T> *>(&initial_space_in);
```

Reply:
Sorry, I somehow missed this. I thought we could not merge anything in libmesh either and did not pay attention to my PR.

Thanks for the fix @jwpeterson

Reply:
No worries, I will merge once everything passes again.